easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = 'PythonBundle'
+
+name = 'rasterio'
+version = '1.2.3'
+
+homepage = 'https://github.com/mapbox/rasterio'
+description = "Rasterio reads and writes geospatial raster data."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),  # for numpy
+    ('matplotlib', '3.3.3'),
+    ('GDAL', '3.2.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('affine', '2.3.0', {
+        'checksums': ['2e045def1aa29e613c42e801a7e10e0b9bacfed1a7c6af0cadf8843530a15102'],
+    }),
+    ('attrs', '20.3.0', {
+        'modulename': 'attr',
+        'checksums': ['832aa3cde19744e49938b91fea06d69ecb9e649c93ba974535d08ad92164f700'],
+    }),
+    ('jmespath', '0.10.0', {
+        'checksums': ['b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9'],
+    }),
+    ('botocore', '1.20.61', {
+        'checksums': ['c765ddd0648e32b375ced8b82bfcc3f8437107278b2d2c73b7da7f41297b5388'],
+    }),
+    ('s3transfer', '0.4.2', {
+        'checksums': ['cb022f4b16551edebbb31a377d3f09600dbada7363d8c5db7976e7f47732e1b2'],
+    }),
+    ('boto3', '1.17.61', {
+        'checksums': ['35b099fa55f5db6e99a92855b9f320736121ae985340adfc73bc46fb443809e9'],
+    }),
+    ('click', '7.1.2', {
+        'checksums': ['d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a'],
+    }),
```

Comment:
Python 3.8.6 provides this version of `click` (https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.8.6-GCCcore-10.2.0.eb#L303) - so this should be removed.

Reply:
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/12791/commits/840bd55e71099051b1e492e176243dca56ca27bb